libs/s25main/desktops/dskCredits.cpp

```diff
@@ -174,16 +290,31 @@ void dskCredits::Draw_()
     DrawCredit();
 }
 
+void dskCredits::GotoNextPage()
+{
+    // Next page
+    ++this->itCurEntry;
+    if(this->itCurEntry == entries.end())
+        this->itCurEntry = entries.begin();
+    timer.restart();
+}
+
+void dskCredits::GotoPrevPage()
+{
+    // Prev page
+    if(this->itCurEntry == entries.begin())
+        this->itCurEntry = std::prev(entries.end());
+    else
+        --this->itCurEntry;
```

Comment:
To mirror the other function:

```suggestion
    if(this->itCurEntry == entries.begin())
        this->itCurEntry = entries.end();
    --this->itCurEntry;
```

BTW: Can you remove the trivial comments? `GotoPrevPage()` --> `// Prev page` is just noise